### PR TITLE
Create initial menu with updates from previous PR. (https://github.com/OSC/bc_access_menu/pull/3)

### PR DESCRIPTION
**Pull request by gerald-byrket**

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (treydock) [inline manifest.yml]: I don't see this file installed in README.md or in RPM spec, so presumably this file isn't needed?

Comment (gerald-byrket) [reply manifest.yml]: That's a great catch.  However, in this scenario, the bc_access_menu repository doesn't need to be copied to /var/www/ood/apps/sys at all.  Only the menu item "apps" need to be copied over.  This is a list of directories in my personal ood instance in /var/www/ood/apps/sys.  The ACCESS menu items show up properly.  You'll notice that bc_access_menu is not "deployed" to /var/ww/ood/apps/sys.

activejobs
bc_access_menu_allocations
bc_access_menu_metrics
bc_access_menu_operations
bc_access_menu_support
bc_desktop
bc_osc_codeserver
dashboard
file-editor
files
myjobs
shell


Here's a screenshot of the menu live.

image

Comment (gerald-byrket) [reply manifest.yml]: i'm going to remove that manifest file.

File: bc_access_menu_allocations/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Allocation
+category: ACCESS
+description: "Allocation\r\n"
+icon: 'fas://magic'
```
Comment (johrstrom) [inline bc_access_menu_allocations/manifest.yml]: We should probably change these icons to something that we at least think will work - I'm sure they'll have feedback on that.

Here's my guess as to which should be what.

metrics -> fas://signal
allocations -> fas://usd
support-> fas://question
operations -> fas://cog

**Review by treydock — APPROVED** [review]

File: bc_access_menu_allocations/manifest.yml [hunk 3]
```diff
@@ -2,6 +2,6 @@
 name: Allocation
 category: ACCESS
 description: "Allocation\r\n"
-icon: 'fas://magic'
+icon: 'fas://usd'
 url: https://allocations.access.org
```
Comment (johrstrom) [inline bc_access_menu_allocations/manifest.yml]:
```suggestion
url: https://allocations.access-ci.org
```

I believe it's `access-ci.org`.

Comment (gerald-byrket) [reply bc_access_menu_allocations/manifest.yml]: These are the links that Andrew provided in the Collaboration Team Running Meeting Notes.

```
Links for ACCESS OnDemand
Operations: https://operations.access.org
Allocation: https://allocations.access.org
Support: https://support.access.org
Metrics: https://metrics.access.org
```

Comment (gerald-byrket) [reply bc_access_menu_allocations/manifest.yml]: We can confirm in today's meeting though.

**Comment by gerald-byrket** [comment]
I've updated the  menus as requested in today's ACCESS meeting.  Here are the menu items requested

<strike>(REMOVE) Operations: https://operations.access-ci.org/ (Not needed?  For Resource Providers?)</strike>
About ACCESS: https://access-ci.org
Allocations: https://allocations.access-ci.org/
Get Support: https://support.access-ci.org
XDMoD Metrics: https://xdmod.access-ci.org/

File: README.md [hunk 4]
```diff
@@ -1,2 +1,50 @@
...
+cd <dir>
+git checkout <tag/branch>
+```
+
+To activate this menu item on your Open OnDemand Portal, you need to:
+```cd bc_access_menu```
+```sudo cp -Rf bc_access_allocations /var/www/ood/apps/sys/```
+```sudo cp -Rf bc_access_metrics /var/www/ood/apps/sys/```
+```sudo cp -Rf bc_access_operations /var/www/ood/apps/sys/```
+```sudo cp -Rf bc_access_support /var/www/ood/apps/sys/```
```
Comment (treydock) [inline README.md]:
```suggestion
cd bc_access_menu
sudo cp -Rf bc_access_menu_about /var/www/ood/apps/sys/
sudo cp -Rf bc_access_menu_allocations /var/www/ood/apps/sys/
sudo cp -Rf bc_access_menu_metrics /var/www/ood/apps/sys/
sudo cp -Rf bc_access_menu_support /var/www/ood/apps/sys/
```

Wrap entire block in one set of triple backticks, not one set per line.  See other sections with code blocks.  Also fixe paths in cp operations to match what actually exists.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,51 @@
...
+cd <dir>
+git checkout <tag/branch>
+```
+
+To activate this menu item on your Open OnDemand Portal, you need to:
+```cd bc_access_menu
+sudo cp -Rf bc_access_about /var/www/ood/apps/sys/
+sudo cp -Rf bc_access_allocations /var/www/ood/apps/sys/
+sudo cp -Rf bc_access_metrics /var/www/ood/apps/sys/
+sudo cp -Rf bc_access_support /var/www/ood/apps/sys/
```
Comment (treydock) [inline README.md]:
```suggestion
sudo cp -Rf bc_access_menu_about /var/www/ood/apps/sys/
sudo cp -Rf bc_access_menu_allocations /var/www/ood/apps/sys/
sudo cp -Rf bc_access_menu_metrics /var/www/ood/apps/sys/
sudo cp -Rf bc_access_menu_support /var/www/ood/apps/sys/
```

**Review by johrstrom — APPROVED** [review]
LGTM

**Comment by gerald-byrket** [comment]
@treydock   Do you have any other comments, or are you good with this now?

**Review by treydock — APPROVED** [review]

**Pull request merged**